src/libs/actions/User.js

```diff
@@ -198,10 +198,21 @@ function getPublicDomainInfo() {
             if (response.jsonCode === 200) {
                 const {isFromPublicDomain} = response;
                 Onyx.merge(ONYXKEYS.USER, {isFromPublicDomain});
+
+                // If the user is not on a public domain we'll want to know whether they are on a domain that has
+                // already provisioned the Expensify card
+                if (isFromPublicDomain) {
+                    return;
+                }
```

Comment:
why do public domains get excluded here?

Reply:
AFAIK only users with private domains as their primary login are eligible for the Expensify card. So, if they are on a public domain we don't need to know if they are using the Expensify card already since we'll show them alternative marketing copy that instructs them update their login first.